dialyzer.ignore-warnings

```diff
@@ -0,0 +1,17 @@
+src/future_elixir_parser.yrl:1108:1: Function return_error/3 has no local return
+src/future_elixir_parser.yrl:1113:1: Function return_error_with_meta/3 has no local return
+src/future_elixir_parser.yrl:1116:1: Function error_invalid_stab/1 has no local return
+src/future_elixir_parser.yrl:1121:1: Function error_bad_atom/1 has no local return
+src/future_elixir_parser.yrl:1126:1: Function bad_keyword/2 has no local return
+src/future_elixir_parser.yrl:1154:1: Function error_no_parens_strict/1 has no local return
+src/future_elixir_parser.yrl:1159:1: Function error_no_parens_many_strict/1 has no local return
+src/future_elixir_parser.yrl:1173:1: Function error_no_parens_container_strict/1 has no local return
+src/future_elixir_parser.yrl:1186:1: Function error_too_many_access_syntax/1 has no local return
+src/future_elixir_parser.yrl:1198:1: Function error_invalid_kw_identifier/1 has no local return
+Unknown function prim_tty:isatty/1
```

Comment:
I think `future_elixir.erl` may be changed more often than `future_elixir_parser.yrl`, so we do not need to specify the file and line number.

Reply:
I'm fine with ignoring all the copied files.